tests/nvapi_sysinfo_topo.cpp

```diff
@@ -347,4 +347,34 @@ TEST_CASE("Topology methods succeed", "[.sysinfo-topo]") {
             REQUIRE(NvAPI_GPU_GetConnectedDisplayIds(gpuHandles[0U], gpu0DisplayIds, &gpu0DisplayIdCount, 0) != NVAPI_INCOMPATIBLE_STRUCT_VERSION);
         }
     }
+
+    SECTION("GetPhysicalGPUs returns OK") {
+        NvPhysicalGpuHandle handles[NVAPI_MAX_LOGICAL_GPUS]{};
+        NvU32 count;
+        REQUIRE(NvAPI_EnumPhysicalGPUs(handles, &count) == NVAPI_OK);
+
+        NV_PHYSICAL_GPUS gpus{};
+        REQUIRE(NvAPI_SYS_GetPhysicalGPUs(&gpus) == NVAPI_OK);
+        REQUIRE(gpus.gpuHandleCount > 0);
+        REQUIRE(gpus.gpuHandleCount == count);
+        for (auto i = 0U; i <= count; i++) {
```

Comment:
Shouldn't we stop before `count`? Or are you checking one-past-end entry if it's zeroed by choice?

Reply:
No, just a bit tired actually ;)